PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudAppPropertyView.java

```diff
@@ -404,15 +416,13 @@ private void triggerPersistentStorage() {
 
         boolean enablePersist = StringUtils.equalsIgnoreCase(text, ENABLE_TEXT);
         if (enablePersist) {
-            Font font = publicUrlHyperLink.getFont();
             if (viewModel.isEnablePersistentStorage()) {
                 renderPersistent(this.viewModel);
             } else {
-                this.persistentLabel.setText("Persistent storage is not available before you save the settings.");
-                persistentLabel.setFont(new Font(font.getName(), Font.ITALIC, font.getSize()));
+                this.persistentLabel.setText("Persistent storage will be displayed here after you save the settings.");
```

Comment:
displayed -> updated? As displayed may make feel storage path is available now but will not be displayed

Reply:
Done